tests/sanity/ignore-2.12.txt

```diff
@@ -0,0 +1,10 @@
+Jenkinsfile shebang
+tests/conjur_variable/test.sh shebang
+tests/conjur_variable/policy/root.yml yamllint:unparsable-with-libyaml
+roles/conjur_host_identity/tests/test.sh shebang
+roles/conjur_host_identity/tests/policy/root.yml yamllint:unparsable-with-libyaml # File loaded by summon utility (in Jenkinsfile), not via Python
```

Comment:
I think the comments for this line and line 10 are swapped. This is a policy file; secrets.yaml is a Summon fiile

Reply:
do we want a comment like this on line 3 too?

Reply:
good catch, swapping comments.